### PR TITLE
Add GTEST macro to institution tests

### LINKED ISSUE #1835: Propagate GTEST updates to Institutions
Some changes to the generic Region tests implemented in #1738 were not propagated to generic Institution tests.  This causes an error when developing tests for new archetypes generated with cycstub.

### PR DESCRIPTION
**Pull request by gonuke**
Fixes #1835

### CODE REVIEW

**Comment by bennibbelink** [comment]
Should the macro also be added to `facility_tests.cc`?

**Comment by gonuke** [comment]
> Should the macro also be added to `facility_tests.cc`?

TL;DR: My first thought was that it wasn't necessary, but I've gone ahead and added it.  

Initially, I thought that most new modules will focus on facilities and therefore it wouldn't be necessary to handle it for facility tests. But your question made me remember that we've created the ability for people to only build Institution archetypes if they like, so best to handle all the variations.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -46,6 +46,7 @@ Since last release
 
 * Removed unnecessary records being added to the Resource database by packaging process (#1761)
 * Removed GTest source code from code coverage reports (#1759)
+* Extended new GTest handling to Institutions as they were done in Regions (#1836)
```
Comment (bennibbelink) [inline CHANGELOG.rst:49]:
```suggestion
* Extended new GTest handling to Institutions and Facilities as they were done in Regions (#1836)
```

**Comment by bennibbelink** [comment]
Made a small suggestion to changelog that you can choose to include if you want.  It is not critical though, and the code changes look good so I am approving

**Review by bennibbelink — APPROVED** [review]

**Comment by bennibbelink** [comment]
Thanks for this quick fix @gonuke

**Pull request merged**